tests/test_jsonshim.py

```diff
@@ -0,0 +1,61 @@
+"""Test postqa/jsonshim."""
+
+from __future__ import (division, absolute_import, print_function,
+                        unicode_literals)
+from builtins import *  # NOQA
+from future.standard_library import install_aliases
+install_aliases()  # NOQA
+
+import os
+import json
+import pkg_resources
+import pytest
+from numpy.testing import assert_approx_equal
```

Comment:
I'm not sure if this was intentional but numpy is not present in `requirements.txt`.